src/main/scala/eventstore/cluster/ClusterProtocol.scala

```diff
@@ -20,22 +20,22 @@ object ClusterProtocol extends DefaultJsonProtocol {
 
   implicit object DateTimeFormat extends JsonFormat[DateTime] {
     val formats = List(
-      JodaFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'"),
-      JodaFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSSSSSS'Z'"),
-      JodaFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss'Z'"),
-      JodaFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSSSS'Z'"),
-      JodaFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSSS'Z'"),
-      JodaFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"),
-      JodaFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SS'Z'"),
-      JodaFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.S'Z'")
+      JodaFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSSSSSZ"),
+      JodaFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSSSSSSZ"),
+      JodaFormat.forPattern("yyyy-MM-dd'T'HH:mm:ssZ"),
```

Comment:
Hi @reinierl should we continue having `JodaFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss'Z'"),` in list  ?